CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # VocalFusion AVS Setup Change Log
 
+## 2.4.0
```

Comment:
Please bump the version to 3.0.0 due to renaming xvf3610 and xvf3615 as xvf3610-int and xvf3615-int.